src/Microsoft.SqlTools.Serialization/HostLoader.cs

```diff
@@ -0,0 +1,82 @@
+﻿//
```

Comment:
This entire file appears to be mostly (entirely?) copy/pasted from the Credentials Service. Can we put it in Hosting in order to share it?

Reply:
Karl suggested that I created a separate hostloader.cs for serialization. It's what we currently do for credentials and servicelayer as well. Only real difference is that each one initializes different services in InitializeRequestHandlersAndServices(). We can probably figure out a way to refactor this to be shared code if we think it's important.